src/Controller/CivicrmController.php

```diff
@@ -56,6 +56,7 @@ public static function create(ContainerInterface $container) {
    * Main controller, passes trough to CiviCRM.
    */
   public function main($args, $extra) {
+    $args = explode('/', $args);
```

Comment:
I think its possible args is not always an array either. May have to check. If array, explode(), if not, do nothing.

Reply:
Well, got an error after applying the patch, but before clearing caches, so checking if array before using explode may not be necessary. People must clear caches after applying the patch

Reply:
Ah that might be why I couldn't reproduce before. I do see it now on the system status page, just not on most pages.

Reply:
This might make it more robust during an upgrade:

```suggestion
    $args = is_array($args) ? $args : explode('/', $args);
```

(I'm not sure if it's true on typical D9 upgrades, but - in many environments, during an upgrade - there will be a period where you have old data but need to navigate to perform admin tasks.)